polaris.shopify.com/src/components/Markdown/components/Colors/Colors.module.scss

```
@@ -0,0 +1,29 @@
+@import '../../../../styles/variables.scss';
+
+.colors {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  gap: var(--p-space-4);
+  margin-bottom: var(--p-space-8);
+  font-size: var(--font-size-75);
+  font-weight: var(--p-font-weight-semibold);
+
+  @media (min-width: $breakpointTablet) {
+    grid-template-columns: repeat(10, 1fr);
+  }
+
+  .colors-swatch {
```

Because this is a CSS Module, we should be able to export the classnames and use them in the JS rather than having global CSS classes:

```suggestion
.Colors {
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  gap: var(--p-space-4);
  margin-bottom: var(--p-space-8);
  font-size: var(--font-size-75);
  font-weight: var(--p-font-weight-semibold);

  @media (min-width: $breakpointTablet) {
    grid-template-columns: repeat(10, 1fr);
  }

  .ColorsSwatch {
```